src/app/clusters/zone-management-server/app_config_dependent_sources.cmake

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the equivalent to app_config_dependent_sources.gni
+TARGET_SOURCES(
+  ${APP_TARGET}
+  PRIVATE
+    "${CLUSTER_DIR}/zone-management-server.cpp"
+    "${CLUSTER_DIR}/zone-management-server.h"
+)
```

Comment:
Unfortunately, the `zone-geometry.h` bit _does_ need to stay in the cmake for now...  We don't have any other build definitions with cmake, sadly.

Reply:
Moved it back.